lib/src/util/io.dart

```diff
@@ -137,36 +137,33 @@ Future<T> getUnusedPort<T>(FutureOr<T> tryPort(int port)) async {
   return value;
 }
 
+/// Whether this computer supports binding to IPv6 addresses.
+bool _supportsIPv6 = null;
+
 /// Returns a port that is probably, but not definitely, not in use.
 ///
 /// This has a built-in race condition: another process may bind this port at
 /// any time after this call has returned. If at all possible, callers should
 /// use [getUnusedPort] instead.
 Future<int> getUnsafeUnusedPort() async {
   var socket;
-  if (await _supportsIPv6) {
-    socket = await RawServerSocket.bind(InternetAddress.LOOPBACK_IP_V6, 0,
-        v6Only: true);
-  } else {
+  if (_supportsIPv6 ?? true) {
```

Comment:
`?? true` skeeves me out a little bit. Maybe make the variable `_maySupportIPv6` and have it default to `true`?